Source/Core/Core/NetPlayServer.cpp

```diff
@@ -42,8 +42,6 @@
 
 namespace NetPlay
 {
-u64 g_netplay_initial_rtc = 1272737767;
```

Comment:
I suppose this number has no real reason to stick around, right? It looks fairly random too (being somewhere in the 1978s)

Reply:
It's in 2010, but yes, it seems arbitrary.

Reply:
Duh. As duration it is somewhere around 40 years, but as timestamp its 2010. Not really useful either way I guess.